modules/trusted-firmware-m/Kconfig

```diff
@@ -233,6 +233,40 @@ config TFM_REGRESSION
 	  When enabled, this option signifies that the TF-M build includes
 	  the Secure and the Non-Secure regression tests.
 
+config TFM_PSA_TEST_CRYPTO
+	bool "Crypto tests"
+	help
+	  Enable the PSA Crypto test suite.
+
+config TFM_PSA_TEST_PROTECTED_STORAGE
+	bool "Storage tests"
+	help
+	  Enable the PSA Protected Storage test suite.
+
+config TFM_PSA_TEST_INTERNAL_TRUSTED_STORAGE
+	bool "Internal Trusted Storage tests"
+	help
+	  Enable the PSA Protected Storage test suite.
+
+config TFM_PSA_TEST_STORAGE
+	bool "Storage tests"
+	help
+	  Enable the PSA Storage test suite. This is a combination of the
```

Comment:
If this is a combination, does it mean that it should "select" the other Kconfig symbols?
And then, these symbols should not have a prompt if _this_ option is enabled?

Reply:
You can only choose one suite at a time anyway. I will change this to a `choice`

Reply:
Thanks, much better.